gulpfile.js

```diff
@@ -16,19 +23,99 @@ const styles = () => {
     .pipe(postcss([
       autoprefixer()
     ]))
+    .pipe(csso())
+    .pipe(rename("styles.min.css"))
     .pipe(sourcemap.write("."))
-    .pipe(gulp.dest("source/css"))
+    .pipe(gulp.dest("build/css"))
     .pipe(sync.stream());
 }
 
 exports.styles = styles;
 
+
+// Images
+
+const images = () => {
+  return gulp.src('source/img/**/*.{jpg,png,svg}')
+  .pipe(imagemin([
+    imagemin.optipng({optimizationLevel: 3}),
+    imagemin.mozjpeg({progressive: true}),
+    imagemin.svgo(),
+  ]))
+  .pipe(gulp.dest("source/img"))
```

Comment:
```suggestion
  .pipe(gulp.dest("build/img"))
```

Сборка должна производиться в ***build**.